README.md

```diff
@@ -1,2 +1,2 @@
 This repository contains all the descriptive materials of the Open Education Hub teaching/learning methodology.
```

Comment:
```suggestion
# Methodology

This repository contains all the descriptive materials of the Open Education Hub teaching/learning methodology.
```
Superlinter requires a top-level header ([MD#041](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md041---first-line-in-a-file-should-be-a-top-level-heading)).